docs/getting_started.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "Getting Started"
+format: html
+---
+
+
+##### Pre-requisites.
+
+1. Docker - Instructions on how to install Docker can be found on the Docker website
+2. Instant OpenHIE v2 CLI - Installation instructions [here](https://jembi.gitbook.io/instant-v2/getting-started/quick-start): 
+
+##### Clone the repository
+
+`git clone https://github.com/IntelliSOFT-Consulting/MamaToto-HIE.git`
```

Comment:
should refer to pharmaccess instance probably a fork of this or repo is transferred